plugins-list.yaml

```diff
@@ -16,80 +17,122 @@ plugins/auth-backend-module-guest-provider:
+# plugins/catalog-backend-module-logs  ==> wrapper
+# plugins/catalog-backend-module-msgraph  ==> wrapper
 plugins/catalog-backend-module-openapi:
 plugins/catalog-backend-module-puppetdb:
 plugins/catalog-backend-module-scaffolder-entity-model:
 plugins/catalog-backend-module-unprocessed:
-#plugins/catalog-graph: ==> Added as a static plugin in RHDH 
-#plugins/catalog-import: ==> Added as a static plugin in RHDH
+plugins/catalog-common:
+# plugins/catalog-graph ==> Added as a static plugin in RHDH :
+# plugins/catalog-import ==> Added as a static plugin in RHDH
+plugins/catalog-node:
+plugins/catalog-react:
 plugins/catalog-unprocessed-entities:
+plugins/catalog-unprocessed-entities-common:
 plugins/config-schema:
 plugins/devtools:
 plugins/devtools-backend:
+plugins/devtools-common:
 plugins/events-backend:
 plugins/events-backend-module-aws-sqs:
 plugins/events-backend-module-azure:
 plugins/events-backend-module-bitbucket-cloud:
 plugins/events-backend-module-gerrit:
-#plugins/events-backend-module-github: # ==> the new backend system is not exported as default export (Error: Backend plugin is not valid for dynamic loading: it should either export a BackendFeature or BackendFeatureFactory as default export, or export a const dynamicPluginInstaller: BackendDynamicPluginInstaller field as dynamic loading entrypoint)
-#plugins/events-backend-module-gitlab: # ==> the new backend system is not exported as default export (Error: Backend plugin is not valid for dynamic loading: it should either export a BackendFeature or BackendFeatureFactory as default export, or export a const dynamicPluginInstaller: BackendDynamicPluginInstaller field as dynamic loading entrypoint)
+# plugins/events-backend-module-github ==> Added as a static plugin in RHDH
+# plugins/events-backend-module-gitlab ==> Added as a static plugin in RHDH
+plugins/events-backend-test-utils:
+plugins/events-node:
 plugins/example-todo-list:
 plugins/example-todo-list-backend:
+plugins/example-todo-list-common:
 plugins/home:
-plugins/kubernetes:
-plugins/kubernetes-backend:
+plugins/home-react:
+# plugins/kubernetes  ==> wrapper
+# plugins/kubernetes-backend  ==> wrapper
+plugins/kubernetes-cluster:
+plugins/kubernetes-common:
+plugins/kubernetes-node:
+plugins/kubernetes-react:
 plugins/notifications:
-plugins/notifications-backend: --embed-package @backstage/plugin-signals-node
+plugins/notifications-backend:
+plugins/notifications-backend-module-email:
+plugins/notifications-common:
+plugins/notifications-node:
 plugins/org:
-#plugins/permission-backend: ==> Added as a static plugin in RHDH
-#plugins/permission-backend-module-policy-allow-all: ==> Not meaningful for RHDH
+plugins/org-react:
+# plugins/permission-backend ==> Added as a static plugin in RHDH
+# plugins/permission-backend-module-policy-allow-all ==> Added as a static plugin in RHDH
+plugins/permission-common:
+plugins/permission-node:
+plugins/permission-react:
 plugins/proxy-backend:
```

Comment:
Isn't it in the RHDH as a static plugin ?

Reply:
I see [corporate-proxy.ts](https://github.com/janus-idp/backstage-showcase/blob/release-1.3/packages/backend/src/corporate-proxy.ts) in show case, is this the replacement for this plugin?

Reply:
no, I don't think so

Reply:
ok, thanks, I see it [here](https://github.com/janus-idp/backstage-showcase/blob/release-1.3/packages/backend/src/index.ts#L83), removing from the list